pkgs/by-name/us/usbkvm/package.nix

```diff
@@ -11,15 +11,16 @@
   pkg-config,
   python3,
   stdenv,
+  udev,
   wrapGAppsHook3,
 }:
 
 let
-  version = "0.1.0";
+  version = "0.2.0";
 
   src = fetchzip {
```

Comment:
why not use fetchFromGitHub?

Reply:
I elaborate on this in the original PR that added this package: #369115.

While I did manage to build this from source, only the release tarballs include pre-built vendored firmware for the USBKVM device. When you build the firmware from source it's not tested and can lead to users soft-bricking their device.

As a result, it seems best we instead use the pre-built and tested release tarballs. This has also been kindly requested by the original author of this package. I have offered to (as an option) also allow this package to be built from the git source, but I'm not sure whether this added complexity in the package definition is warranted.